plugins/chat/app/controllers/chat/chat_controller.rb

```diff
@@ -176,6 +176,7 @@ def messages
       messages = preloaded_chat_message_query.where(chat_channel: @chat_channel)
       messages = messages.with_deleted if guardian.can_moderate_chat?(@chatable)
       messages = messages.where(thread_id: params[:thread_id]) if params[:thread_id]
+      messages = exclude_thread_messages(messages) if should_exclude_thread_messages?
```

Comment:
Nit pick, feel free to ignore, but I would probably go with `!include_thread_messages?` as method here

Reply:
Yep fair enough!